spec/models/ingest/converter_spec.rb

```diff
@@ -151,28 +146,6 @@
     end
   end
 
-  describe '.total_row_count' do
-    it 'returns the total number of rows contained in the Excel file' do
-      expect(converter.total_row_count).to eql 3
-    end
-
-    context 'with a download that has only one sheet to convert' do
-      let(:download) { fake_download('rm3767-with-no-orders.xls') }
-
-      it 'returns the total number of rows contained in the Excel file' do
-        expect(converter.total_row_count).to eql 1
-      end
-    end
-
-    context 'with a download that contains rows containing only whitespace' do
-      let(:download) { fake_download('rm3767-with-whitespace-rows.xls') }
-
-      it 'returns the total number of filled rows contained in the Excel file' do
-        expect(converter.total_row_count).to eql 2
-      end
-    end
-  end
```

Comment:
This is no longer the responsibility of the converter but we haven't got an explicit `orchestrator_spec.rb` example for it; running the converter (either through the orchestrator or through this spec) is a relatively expensive thing so rather than add a spec in there just for whitespace we've quietly dropped an explicit spec for whitespace. Unsure as to whether this is covered elsewhere.